client/package.json

```diff
@@ -9,21 +9,24 @@
         "ajv": "^6.12.0",
         "babel-polyfill": "^6.26.0",
         "cross-os": "^1.3.0",
+        "debug": "^4.1.1",
         "form-data": "^3.0.0",
         "js-yaml": "^3.13.1",
         "react": "^16.13.0",
         "react-ace": "^8.0.0",
         "react-dom": "^16.13.0",
         "react-scripts": "3.4.0",
+        "regenerator-runtime": "^0.13.5",
         "url": "^0.11.0"
     },
     "scripts": {
-        "start": "react-scripts start",
+        "start": "PORT=3000 react-scripts start",
         "build": "cross-os cross-os-build",
         "eject": "react-scripts eject",
         "test": "react-scripts test",
         "posttest": "license-checker --summary",
-        "report-coverage": "nyc report --reporter=text-lcov > coverage.lcov"
+        "report-coverage": "nyc report --reporter=text-lcov > coverage.lcov",
+        "unit-test": "npm test -- --coverage"
```

Comment:
Why not add it to the test script?

Reply:
adding the `--coverage` flag makes the unit tests run a lot slower, which can get annoying while developing + testing so I made another script that runs tests with coverage. Any suggestions for something that makes more sense?

I still need to figure out how to make a report of the code coverage produced by jest -- coverage. there's a configuration for it. we can explore this in a different PR